<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget1" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,1">
      <item>
       <widget class="QPushButton" name="button1">
        <property name="text">
         <string>one</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button2">
        <property name="text">
         <string>two</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button3">
        <property name="text">
         <string>three</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget2" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="button4">
        <property name="text">
         <string>four</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button5">
        <property name="text">
         <string>five</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button6">
        <property name="text">
         <string>six</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget3" native="true">
     <widget class="QPushButton" name="button7">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>seven</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button8">
      <property name="text">
       <string>eight</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button9">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>40</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>nine</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
